app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -30,14 +30,14 @@ class VariantManagerTest {
     // SERP Experiment(s)
 
     @Test
-    fun serpControlVariantIsInactiveAndHasNoFeatures() {
+    fun serpControlVariantIsActiveAndHasNoFeatures() {
```

Comment:
I'm reverting this back to the original values, it's not this PR responsibility to change it

Reply:
Agree, but what about removing the experiment for you to get more users into your experiment?

Reply:
as discussed in the call we'll get rid of it in a separate PR